### PR TITLE
Fix/avoid sagify module import clash

### PR DESCRIPTION
**Pull request by pm3310**
Why?
`sagify init` automatically creates a module called `sagify`. However, we have specific methods from sagify library such as `from sagify.api.hyperparameter_tuning import log_metric`. Python interpreter gets confused if it tries to load `from sagify.api.hyperparameter_tuning import log_metric`. It think it should search under the automatically created module `sagify`, then, it cannot find and raises an error. This PR fixes this issue.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,26 +15,26 @@
...
+            'template/sagify_base/local_test/test_dir/output/.gitkeep',
+            'template/sagify_base/local_test/test_dir/model/.gitkeep',
+            'template/sagify_base/local_test/test_dir/input/config/*.json',
+            'template/sagify_base/local_test/test_dir/input/data/training/'
             '.gitkeep'
         ]
     },
     install_requires=[
-        'boto3==1.9.220',
+        'boto3',
```
Comment (ilazakis) [inline setup.py]: Any specific reason for removing the pinned version of `boto3` in particular?

Comment (pm3310) [reply setup.py]: Hey @ilazakis , users have to use awscli that depends on boto3. Each end user has a different version of awscli and by specifying a specific boto3 version may cause conflict.

File: tests/commands/test_configure.py [hunk 2]
```diff
@@ -1,59 +1,59 @@
-from collections import namedtuple
```
Comment (ilazakis) [inline tests/commands/test_configure.py]: @pm3310 Missed this when reviewing. Did these tests need to be disabled? Can we re-enable them? Also, looks like the 0.20.0 merge failed on _Python3_.

Comment (pm3310) [reply tests/commands/test_configure.py]: @ilazakis good catch man! My bad :-( I'll create a new PR for this one

**Pull request merged**